### DISCUSSION TITLE
Help me, want the subs and videos separate

### DISCUSSION THREAD

**Discussion by TheSilentKiller07** [anidl/multi-downloader-nx#742] [Q&A]
Hi folks, 
1st, Thanks for the prog ♥ 
2nd, I only want to know how to stop that mkvmerge from muxing the vid and subs together at the end, so I can get the video files and subtitles separately!
Best regards.

**Comment by Jaynator495** [comment]
--skipSubMux

**Reply by TheSilentKiller07** [reply]
i found that in C:\Users\Yasuo\Desktop\crnch\gui\server\build\modules\module.args 
but what should i do here:
{
        name: 'skipSubMux',
        describe: 'Skip muxing the subtitles',
        docDescribe: true,
        group: 'mux',
        service: ['all'],
        type: 'boolean',
        usage: '',
        default: {
            default: false
        }

**Reply by Jaynator495** [reply]
Nothing? That's if you wanted to edit the flag itself, if you want it to be default you can put it in ./config/cli-defaults.yml like this:
```yaml
skipSubMux: true
```

**Reply by TheSilentKiller07** [reply]
Thanks sir it worked.
